shared/AppInsightsCore/src/JavaScriptSDK/BaseTelemetryPlugin.ts

```diff
@@ -158,6 +161,8 @@ export abstract class BaseTelemetryPlugin implements ITelemetryPlugin {
                     }
                 }
             };
+
+            proxyFunctionAs(_self, "_addUnloadCb", () => _unloadHandlerContainer, "add");
```

Comment:
This exposes effectively the same function as
```typescript
_self._addUnloadCb = (<function arguments>) => {
    return _unloadHandlerContainer.add(<function arguments>);
};
```

The reason it's using a function to return the _unloadHandlerContainer is to support the value of this being changed without having to redefine the prototype function